### PR TITLE
Create a Recommendation File for Cincinnati

### PR DESCRIPTION
**Pull request by mbendure**
Things to Do:

- [x] Add commit
- [x] Review commit
- [ ] Official review
- [ ] Merge

### CODE REVIEW

**Comment by limeri** [comment]
In your things to-do list, if you put a space between the "[]"'s, you'll get check boxes.  :1st_place_medal:

File: mbendure-Cincinnati.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- City View
+- Eagle OTR
+- Bakersfield
+
+### Places to Visit
+- Jungle Jim's
+- Know Theater
+
+### Places to Avoid
+- Great American Tower
```
Comment (limeri) [inline mbendure-Cincinnati.md:13]: Maybe the "(not-so) Great American Tower" 😃

**Comment by limeri** [comment]
Sorry, I think you need a space between the "-[" as well.

**Review by limeri — APPROVED** [review]

**Comment by mbendure** [comment]
Thanks for the tips on getting the check boxes :+1:

**Pull request merged**